### DISCUSSION TITLE
TypeError: this.popComp.popper.updatePosition is not a function

### DISCUSSION THREAD

**Discussion by rpauschert** [sa-si-dev/virtual-select#150] [General]
Hi,
I like your virtual-select very much.
But if I set the option showValueAsTags: true, I get  "TypeError: this.popComp.popper.updatePosition is not a function"

After that I tried the version virtual-select-1.0.27.min.js
In this version there is no Type error. 

Could you please check the issue?
Thanks a lot
Roland

**Comment by sa-si-dev** [comment]
@rpauschert Please share the code you are using to initiate the plugin and the plugin version that you are getting error on.
